### PR TITLE
[zh] Sync changes to Hello Minikube page

### PR DESCRIPTION
**Pull request by tengqm**
This is a sync of English site change (236c65c18e71fbc0d62adf6c0bbe7ded74676fec).

### CODE REVIEW

File: content/zh/docs/tutorials/hello-minikube.md [hunk 1]
```diff
@@ -22,64 +21,48 @@ menu:
...
 ## {{% heading "objectives" %}}
 
-
 <!--
 * Deploy a hello world application to Minikube.
 * Run the app.
 * View application logs.
 -->
-* 将 "Hello World" 应用程序部署到 Minikube。
+* 将一个示例应用部署到 Minikube。
```
Comment (xichengliudui) [inline content/zh/docs/tutorials/hello-minikube.md:55]: 原本翻译并没有问题, 更新是有什么特殊目的吗?

File: content/zh/docs/tutorials/hello-minikube.md [hunk 2]
```diff
@@ -22,64 +21,48 @@ menu:
...
+如果你已在本地安装 [Minikube](/zh/docs/tasks/tools/install-minikube/)，也可以按照本教程操作。
 {{< /note >}}
 
 
-
 ## {{% heading "objectives" %}}
 
-
 <!--
 * Deploy a hello world application to Minikube.
```
Comment (tanjunchen) [inline content/zh/docs/tutorials/hello-minikube.md:51]: Deploy a sample application to Minikube.

File: content/zh/docs/tutorials/hello-minikube.md (hunk 1 above)
Comment (tanjunchen) [reply content/zh/docs/tutorials/hello-minikube.md:55]: see https://github.com/kubernetes/website/pull/24465/files#diff-6086b884d55865ab09248ee626122302R24, 可能没有同步更新英文吧

File: content/zh/docs/tutorials/hello-minikube.md (hunk 2 above)
Comment (tanjunchen) [reply content/zh/docs/tutorials/hello-minikube.md:51]:
```
Deploy a sample application to Minikube.
Run the app.
View application logs.
```

**Review by tanjunchen — COMMENTED** [review]
/approve
we need to sync en comment.

File: content/zh/docs/tutorials/hello-minikube.md (hunk 1 above)
Comment (tengqm) [reply content/zh/docs/tutorials/hello-minikube.md:55]: Right. English text wasn't synced. Will fix.

File: content/zh/docs/tutorials/hello-minikube.md (hunk 2 above)
Comment (tengqm) [reply content/zh/docs/tutorials/hello-minikube.md:51]: yes. Fixed.

**Comment by tengqm** [comment]
As for the reason of the upstream change, pls see commit message https://github.com/kubernetes/website/commit/236c65c18e71fbc0d62adf6c0bbe7ded74676fec.

**Pull request merged**